utils/utils_test.go

```diff
@@ -23,13 +23,10 @@ func TestStringArrayIntersection(t *testing.T) {
 		"def",
 	}
 
-	if len(StringArrayIntersection(a, b)) != 0 {
-		t.Fatal("should be 0")
-	}
+	assert := assert.New(t)
 
-	if len(StringArrayIntersection(a, c)) != 1 {
-		t.Fatal("should be 1")
-	}
+	assert.Equal(len(StringArrayIntersection(a, b)), 0)
```

Comment:
We prefer here to use `assert.Len` to be more semantic.

Reply:
Ok